docs/backplane/requirements/acs/10-acs-managed-service.md

```diff
@@ -32,5 +43,38 @@ Permissions at cluster scope.
 * get pods and pod logs
 * view deployments
 * get deployments
+* view routes
+* view egress firewalls
+* view roles and rolebindings
+* patch persistentvolumeclaims
+
+ACS team needs to inspect the `cloud-service-sensible-declarative-configs` secret
+
+* view secret `cloud-service-sensible-declarative-configs`
+
+ACS team needs to be able to delete a namespace
+
+* delete namespaces
+
+## backplane-acs-ingress: `openshift-ingress`
+
+- ACS team needs get/list/watch access to inspect the routers and logs
+
+* view deployments
+* view pods
+* view pod logs
+
+## backplane-acs-monitoring: `openshift-monitoring`
+
+- ACS team needs get/list/watch access to the openshift monitoring prometheus-related pods, statefulsets and services
+
+* view pods
+* view pods/log
+* view statefulsets
+* view services
+
+- ACS team needs to be able to increase the memory of prometheus
```

Comment:
This is managed from the Prometheus CR which in turn is managed from cluster-monitoring-config ConfigMap which is managed in MCC.  Manual changes should not be made in-cluster.

Reply:
Woops. Wrong namespace, It should be in a `rhacs-observability` namespace role. will update